src/index.ts

```diff
@@ -0,0 +1,4 @@
+// This is an entrypoint that mirrors the interface that the `netlify` package
```

Comment:
thought: Should we log a warning here that the package has been renamed?

Reply:
That feels a bit extreme? We might be logging something that is not actionable to the user, since this might be a transitive dependency.